HW21/tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from loguru import logger
+import sys
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--log-level-custom",
+        default="INFO",
+        choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
+    )
+
+
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
You do not need to use `session` with `autouse`, it will be used automatically. If I'm mistaken
```suggestion
@pytest.fixture(scope="session")
```

Reply:
It doesn't work that way for me :( If I remove "autouse=True", then I have to explicitly call the fixture.